tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,321 @@
+
+> The default security mechanisms within the software produced by the project MUST NOT depend on cryptographic algorithms or modes with known serious weaknesses (e.g., the SHA-1 cryptographic hash algorithm or the CBC mode in SSH).
+
+_Possible answers: Met/Unmet/NA_
+
+> The project SHOULD support multiple cryptographic algorithms, so users can quickly switch if one is broken. Common symmetric key algorithms include AES, Twofish, and Serpent. Common cryptographic hash algorithm alternatives include SHA-2 (including SHA-224, SHA-256, SHA-384 AND SHA-512) and SHA-3.
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST support storing authentication credentials (such as passwords and dynamic tokens) and private cryptographic keys in files that are separate from other information (such as configuration files, databases, and logs), and permit users to update and replace them without code recompilation. If the project never processes authentication credentials and private cryptographic keys, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+> The software produced by the project SHOULD support secure protocols for all of its network communications, such as SSHv2 or later, TLS1.2 or later (HTTPS), IPsec, SFTP, and SNMPv3. Insecure protocols such as FTP, HTTP, telnet, SSLv3 or earlier, and SSHv1 SHOULD be disabled by default, and only enabled if the user specifically configures it. If the software produced by the project does not support network communications, select "not applicable" (N/A). 
+
+_Possible answers: Met/Unmet/NA_
+
+> The software produced by the project SHOULD, if it supports or uses TLS, support at least TLS version 1.2. Note that the predecessor of TLS was called SSL. If the software does not use TLS, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+> The software produced by the project MUST, if it supports TLS, perform TLS certificate verification by default when using TLS, including on subresources. If the software does not use TLS, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+> The software produced by the project MUST, if it supports TLS, perform certificate verification before sending HTTP headers with private information (such as secure cookies). If the software does not use TLS, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+## Secure release
+
+> The project MUST cryptographically sign releases of the project results intended for widespread use, and there MUST be a documented process explaining to users how they can obtain the public signing keys and verify the signature(s). The private key for these signature(s) MUST NOT be on site(s) used to directly distribute the software to the public. If releases are not intended for widespread use, select "not applicable" (N/A).
```

Comment:
Met

Reply:
Resolved in 31a69d5b5c4e7b3061bfd2ba292c006f793cf69c

Reply:
It may be worth clarifying some nuance here. The project only signs the [`SHASUMS.txt`](https://nodejs.org/dist/latest-v20.x/SHASUMS256.txt) file rather than signing each of the release assets.